src/operator/nn/batch_norm-inl.h

```diff
@@ -56,7 +56,7 @@ constexpr int DEFAULT_AXIS = 1;
 }  // namespace batchnorm
 
 /*! \brief Parameters for BatchNoram operator */
-struct BatchNormParam : public dmlc::Parameter<BatchNormParam> {
+struct MXNET_API BatchNormParam : public dmlc::Parameter<BatchNormParam> {
```

Comment:
I don't see MXNET_API used outside of main mcnet include directory.  Why have it internally like this?

Reply:
Ok, I get it now -- for the unit tests. I think better to link statically?